src/parser.cpp

```diff
@@ -479,6 +479,168 @@ options_metadata parser::load_settings()
+        "The retargeting inverval in amount of blocks, defaults to 2016."
+    )
+    (
+        "bitcoin.genesis_block",
+        value<config::block>(&configured.bitcoin.genesis_block),
+        "The genesis block in hex, defaults to Satoshi's genesis block."
+    )
+    (
+        "bitcoin.first_version",
+        value<uint32_t>(&configured.bitcoin.first_version),
+        "The first block version, defaults to 1."
+    )
+    (
+        "bitcoin.bip34_version",
+        value<uint32_t>(&configured.bitcoin.bip34_version),
+        "The bip34 block version, defaults to 2."
+    )
+    (
+        "bitcoin.bip66_version",
+        value<uint32_t>(&configured.bitcoin.bip66_version),
+        "The bip66 block version, defaults to 3."
+    )
+    (
+        "bitcoin.bip65_version",
+        value<uint32_t>(&configured.bitcoin.bip65_version),
+        "The bip65 block version, defaults to 4."
+    )
+    (
+        "bitcoin.bip9_version_bit0",
+        value<uint32_t>(&configured.bitcoin.bip9_version_bit0),
+        "The bip9 version bit 0, defaults to 1."
+    )
+    (
+        "bitcoin.bip9_version_bit1",
+        value<uint32_t>(&configured.bitcoin.bip9_version_bit1),
+        "The bip9 version bit 1, defaults to 2."
+    )
+    (
+        "bitcoin.bip9_version_base",
+        value<uint32_t>(&configured.bitcoin.bip9_version_base),
+        "The bip9 version base, defaults to 536870912."
+    )
+    (
+        "bitcoin.net_active",
+        value<size_t>(&configured.bitcoin.net_active),
+        "The amount of new version blocks needed to activate a softfork, defaults to 750."
+    )
+    (
+        "bitcoin.net_enforce",
+        value<size_t>(&configured.bitcoin.net_enforce),
+        "The amount of new version blocks needed to enforce a softfork, defaults to 950."
+    )
+    (
+        "bitcoin.net_sample",
+        value<size_t>(&configured.bitcoin.net_enforce),
+        "The total amount of blocks considered for a softfork, defaults to 1000."
+    )
+    (
+        "bitcoin.bip65_freeze",
+        value<size_t>(&configured.bitcoin.bip65_freeze),
+        "The block height to freeze the bip65 softfork as in bip90, defaults to 388381."
+    )
+    (
+        "bitcoin.bip66_freeze",
+        value<size_t>(&configured.bitcoin.bip66_freeze),
+        "The block height to freeze the bip66 softfork as in bip90, defaults to 363725."
+    )
+    (
+        "bitcoin.bip34_freeze",
+        value<size_t>(&configured.bitcoin.bip34_freeze),
+        "The block height to freeze the bip34 softfork as in bip90, defaults to 227931."
+    )
+    (
+        "bitcoin.bip16_activation_time",
+        value<uint32_t>(&configured.bitcoin.bip16_activation_time),
+        "The activation time for bip16 in unix time, defaults to 1333238400."
+    )
+    (
+        "bitcoin.bip34_active_checkpoint",
+        value<config::checkpoint>(&configured.bitcoin.bip34_active_checkpoint),
+        "The hash:height checkpoint for bip34 activation."
```

Comment:
Comments should include default values (and below).

Reply:
Done. I see room for improvement here. The settings classes already know the default values; there should be no need to add these values manually.

Reply:
Details are sometimes different in the node or server, so it’s more complex than that. Possible, but IMO not worth the effort.

Reply:
I see.